### ISSUE TITLE
Dependency on deprecated sassc

### ISSUE THREAD

**Issue by habernal** [TracksApp/tracks#2693]
`sassc-ruby` is deprecated ( https://github.com/sass/sassc-ruby/issues/220 ) and unmaintained for some 2-3 years now. Don't know what the package does, but it's a nasty one, as it takes forever to compile when installing Tracks ( https://github.com/sass/sassc-ruby/issues/189 ).

**Comment by habernal** [comment]
I had to fix the neverending compilation of sassc 2.4.0 by changing the Gemfile AND Gemfile.lock to use exactly sassc 2.1.0, otherwise the installation would never finish. Tracks release 2.5.2, Ubuntu 20.04 LTS, AWS with 1GB RAM.

**Comment by ZeiP** [comment]
Yep, this should be updated. I'm not a big frontend expert so this isn't really up my alley, so any help is greatly appreciated. The current Docker images should work ok and don't need a separate compilation, so that's one way to get an environment up easy and fast.
